db/migrate/20240912170756_create_cohorts.rb

```diff
@@ -1,13 +1,13 @@
 class CreateCohorts < ActiveRecord::Migration[7.1]
   def change
     create_table :cohorts do |t|
-      t.integer :program_id
+      t.references :program_id, null: false, foreign_key: { to_table: :programs }
```

Comment:
I agree with Reza you don't need to use t.references when the column and the table are using the same name because it knows already.

I am not sure if it will affect the code later or not thought, but it looks like it was fixed later in other migrations.